plextraktsync/media.py

```diff
@@ -27,10 +27,8 @@ def __init__(
     ):
         self.plex_api = plex_api
         self.trakt_api = trakt_api
-        self.mf = mf
```

Comment:
you must get rid of the constructor parameter too if you don't assign it in constructor